prov/efa/src/rxr/rxr_ep.c

```diff
@@ -1452,7 +1454,8 @@ int rxr_ep_init(struct rxr_ep *ep)
 		 */
 		ret = rxr_create_pkt_pool(ep, entry_sz,
 					  rxr_env.readcopy_pool_size,
-					  0, &ep->rx_readcopy_pkt_pool);
+					  flags & ~OFI_BUFPOOL_HUGEPAGES,
```

Comment:
use `pkt_pool_flags` to be consistent. I understand it changed behavior to use huge pages when fork support is off, which is ok.